utils/search/search_general.py

```diff
@@ -386,17 +404,17 @@ def search_prepare_query(search_query,
     return query
 
 
-def remove_facet_filters(filter_query):
+def remove_facet_filters(parsed_filters):
     """Process query filter string to keep only non facet filters
 
     Useful for being able to combine classic facet filters and clustering. Addtionaly, it returns
```

Comment:
Maybe add some explanation of what a "facet filter" is? I assume these are the filters displayed as facets in the search interface. Maybe just adding this info would be enough. Also give an intuition about why this is "Useful for being able to combine classic facet filters and clustering"